projects/ngx-prx-styleguide/src/lib/tz-datepicker/tz-datepicker.component.css

```diff
@@ -1,6 +1,16 @@
-input {
+:host > * {
   margin: 5px 0;
-  height: 40px;
+}
```

Comment:
The tz datepicker inputs are a little more compact now but with added margin. The container element in the Publish box is flex column aligned to start so the inputs stack; otherwise, date and time elements can lay out inline by default for larger areas.